candidate_word.h

```diff
@@ -20,9 +20,7 @@
 #ifndef __CANDIDATE_WORD_H__
 #define __CANDIDATE_WORD_H__
 
-#include <QDeclarativeView>
-#include <QObject>
-#include <qdeclarative.h>
```

Comment:
That's also my guess. I remember seeing it somewhere in a porting guide but could not find it anymore. It also makes it easier to support both Qt4 and Qt5.